extension/test/app/cloudwatch-widget.test.ts

```diff
@@ -1,41 +1,73 @@
-import { AdoMock } from '../helpers/ado-mock';
-const adoMock = AdoMock.initMock();
+import { AdoApiMock, AdoSdkMock, AdoServiceEndpointMock } from '../helpers';
+const sdkMock = AdoSdkMock.initMock();
```

Comment:
I hate this stuff but apparently the azure stuff is built with amd and jest does not like that shit, so to work around you have to mock it before you import files that reference the amd modules

Reply:
> I hate this stuff but apparently the azure stuff is built with amd and jest does not like that shit

:rofl: no worries, have had to use similar types of mini hacks before to get some nice tests in place. always worth the effort in my experience